agent/security_helper.py

```diff
@@ -33,3 +35,18 @@ def is_firewall_enabled():
     filter_table = iptc.Table(iptc.Table.FILTER)
     input_chain = next(filter(lambda c: c.name == 'INPUT', filter_table))
     return not len(input_chain.rules) == 0
+
+
+def netstat_scan():
+    """
+    Returns all open inet connections with their addresses and PIDs.
+    """
+    connections = psutil.net_connections(kind='inet')
```

Comment:
Shouldn't we user `all` instaed of `inet`. It isn't however clear what the difference is in the manual at a quick glance

Reply:
@vpetersson `all` also gathers UNIX sockets. We don't need them, I suppose.

Reply:
Ah ok - yeah we don't need that for now as we primarily focus on remote attack vectors.